test/e2e/storage/pvc_protection.go

```diff
@@ -63,7 +65,7 @@ var _ = utils.SIGDescribe("PVC Protection", func() {
 		By("Checking that PVC Protection finalizer is set")
 		pvc, err = client.CoreV1().PersistentVolumeClaims(pvc.Namespace).Get(pvc.Name, metav1.GetOptions{})
 		Expect(err).NotTo(HaveOccurred(), "While getting PVC status")
-		Expect(slice.ContainsString(pvc.ObjectMeta.Finalizers, volumeutil.PVCProtectionFinalizer, nil)).To(BeTrue())
+		Expect(slice.ContainsString(pvc.ObjectMeta.Finalizers, volumeutil.PVCProtectionFinalizer, nil)).To(BeTrue(), fmt.Sprintf("PVC Protection finalizer(%v) is not set in %v", volumeutil.PVCProtectionFinalizer, pvc.ObjectMeta.Finalizers))
```

Comment:
IIRC there is no need to use `fmt.Sprintf` here. `Expect` already supports a variadic/printf style of formatting messages.

Reply:
Thanks for your advice, I see.
I will try it on next update :-)